### PR TITLE
Add application version to "fix" out put when verbosity flag is set

### PR DESCRIPTION
**Pull request by SpacePossum**
closes https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/5358

### CODE REVIEW

**Review by keradus — APPROVED** [review]

File: src/Console/Command/FixCommand.php [hunk 1]
```diff
@@ -156,6 +156,11 @@ protected function execute(InputInterface $input, OutputInterface $output)
         ;
 
         if (null !== $stdErr) {
+            if (OutputInterface::VERBOSITY_VERBOSE <= $output->getVerbosity()) {
```
Comment (keradus) [inline src/Console/Command/FixCommand.php:159]: maybe we should add it to all commands? (of course only for verbose output and only to stdErr)
eg, `describe` output is also different for different PHP CS Fixer version.

Comment (SpacePossum) [reply src/Console/Command/FixCommand.php]: :+1:

File: tests/Console/Command/SelfUpdateCommandTest.php [hunk 2]
```diff
@@ -391,7 +392,7 @@ private function createToolInfo($isInstalledAsPhar = true)
 
     private function getToolPath()
     {
-        return "{$this->root->url()}/php-cs-fixer";
+        return "{$this->root->url()}/PHP CS Fixer";
```
Comment (glensc) [inline tests/Console/Command/SelfUpdateCommandTest.php:395]: too greedy replacement here?

ps: if this was my project, I would say such changeset needs to be in separate PR. too many logical changes in a single PR is never good. should be 1 PR per 1 feature. also each change should be one commit, so it would be easier to remove commit if it's rejected by a review.

Comment (glensc) [reply tests/Console/Command/SelfUpdateCommandTest.php:395]: and commits should be atomic:
- https://www.freshconsulting.com/atomic-commits/

so change commit like "fix tests" would never exist, you "fix tests" in the commit that broke it. such fixup based flow I have found this helper very useful:
- https://github.com/keis/git-fixup

Comment (SpacePossum) [reply tests/Console/Command/SelfUpdateCommandTest.php:395]: yes indeed, nice catch :+1:

Comment (SpacePossum) [reply tests/Console/Command/SelfUpdateCommandTest.php:395]: about the amount of changes and commits and PR's, I agree, yet I balance against velocity and being practical as well as there is a lot people want from this project and only a limited amount of time to do it

Comment (glensc) [reply tests/Console/Command/SelfUpdateCommandTest.php:395]: @SpacePossum in this specific case, if you would stage that "Fix tests" into main commit (that introduced the breakage), and add it as `git fixup`, and later `git rebase -i --autosquash`, it's very easy to extract commits and move them to separate branch (and PR).

so it's already a different kind of development model, where you fix commits, not add and add commits.

for developing (and checking CI), it's easier if the commits are in a single branch.
